CHANGES.md

```diff
@@ -1,16 +1,18 @@
 ## 1.3.5 2018-01-17
 
+This release focuses on improvements and ixes for Namerd and the Linkerd Admin UI, and features a number of community contributions!
+
 * 🎓 H2 router and `io.l5d.mesh` Namerd interface are no longer experimental ([#1782](https://github.com/linkerd/linkerd/pull/1782))! 🎓
 * Add an experimental namer for Rancher service discovery ([#1740](https://github.com/linkerd/linkerd/pull/1740)). A huge thank you to [@fangel](https://github.com/fangel) for contributing this namer!
 * Kubernetes
   * Fix a bug that could cause the `io.l5d.k8s` namer to get "stuck" and fail to recieve updates from an endpoint ([#1755](https://github.com/linkerd/linkerd/pull/1755)). Contributed by [@obeattie](https://github.com/obeattie).
 * Admin UI
-  * Add `/admin/client_state.json` endpoint to return the current address set of each client ([#1768](https://github.com/linkerd/linkerd/pull/1768)).
+  * Add `/admin/client_state.json` endpoint to return the current address set of each client to aid in debugging ([#1768](https://github.com/linkerd/linkerd/pull/1768)).
```

Comment:
Maybe: Add an `/admin/client_state.json` debugging endpoint to expose the current address set of each client, allowing you to easily inspect where Linkerd thinks it can send traffic to.